api-module-library/slack/models/integrationMapping.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const mongoose = require('mongoose');
+const { IntegrationMapping: Parent } = require('@friggframework/module-plugin');
+
+const schema = new mongoose.Schema({});
+
+const name = 'SlackMessage';
```

Comment:
I think this one belongs in your own frigg instance (ironclad-frigg), local model there.

Reply:
I've got other thoughts on the models to build themselves (specifically, thinking this would be 'WorkflowConfig' or something to that effect), but yeah, we can put those thoughts over there.

Reply:
... On second thought, it probably makes sense to have WorkflowConfig be available in this module as an exported model with default schema, since any external partner/dev that uses this module would likely also want to store information about a given workflow and not have them be attached directly to the integration config.